src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/routing.yml

```
@@ -34,7 +34,7 @@ akeneo_connectivity_connection_error_management_dashboard:
    path: '/connect/data-flows/business-errors'
 
 akeneo_connectivity_connection_error_management_connection_monitoring:
-   path: '/connections/{code}/monitoring'
+   path: '/connect/connection-settings/{code}/monitoring'
```

```suggestion
   path: '/connect/connections/{code}/monitoring'
```
For all the URLs, I would keep the `connections` path as many pages have nothing to do with the settings.

If we want to be more precise for the ones that do, something like `connections/{code}/settings/event-subscription` (as un example) would be better.

---

I have raised my point-of-view in the story, no need to do it here :)